### ISSUE TITLE
speedtest cli failing on RaspberryPi

### ISSUE THREAD

**Issue by VALarry** [Tysonpower/starlinkstatus#39]
speedtest --version
speedtest-cli 2.1.3
Python 3.9.2 (default, Mar 12 2021, 04:06:34) [GCC 10.2.1 20210110]

usage: speedtest [-h] [--no-download] [--no-upload] [--single] [--bytes]
                 [--share] [--simple] [--csv] [--csv-delimiter CSV_DELIMITER]
                 [--csv-header] [--json] [--list] [--server SERVER]
                 [--exclude EXCLUDE] [--mini MINI] [--source SOURCE]
                 [--timeout TIMEOUT] [--secure] [--no-pre-allocate]
                 [--version]
speedtest: error: unrecognized arguments: -V --accept-license --accept-gdpr
speedtest is running...
usage: speedtest [-h] [--no-download] [--no-upload] [--single] [--bytes]
                 [--share] [--simple] [--csv] [--csv-delimiter CSV_DELIMITER]
                 [--csv-header] [--json] [--list] [--server SERVER]
                 [--exclude EXCLUDE] [--mini MINI] [--source SOURCE]
                 [--timeout TIMEOUT] [--secure] [--no-pre-allocate]
                 [--version]
speedtest: error: unrecognized arguments: -f json

Dropping the bad arguments allows the script to finish, but no speedtest data is captured to transmit (both 0 mbps).  Was working until 10/29 so I bet apt had an update to Speedtest that busted it, which means it is coming for everyone eventually.

**Comment by Tysonpower** [comment]
nope, all fine (i really was shocked thinking they removed the json format) you just installed the wrong speedtest-cli :)

please remove all non official speedtest cli versions and get the original one from ookla:
https://www.speedtest.net/de/apps/cli
